src/lib/tradingView/utils.ts

```diff
@@ -8,6 +8,8 @@ import { Themes } from '@/styles/themes';
 
 import { AppTheme, type AppColorMode } from '@/state/configs';
 
+const MIN_NUM_TRADES_FOR_ORDERBOOK_PRICES = 10;
```

Comment:
[non-blocking] this function is a great opportunity for a small unit test 😄

Reply:
🤮